src/test/scala/com/mwz/sonar/scala/metadata/package.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2018-2020  All sonar-scala contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.mwz.sonar.scala
+
+import cats.kernel.Eq
+
+package object metadata {
+  implicit val eqRule: Eq[Rule] = Eq.fromUniversalEquals
```

Comment:
Wouldn't it would be better to put this in the companion object of `Rule`.

Reply:
For some reason, I couldn't get it picked up from the companion object, but it turns out this isn't even needed any more. 🤷‍♂